ccd

```diff
@@ -125,14 +125,21 @@ function clean_name() {
 function git_clone() {
     project=$1
     branch=$2
+    local_repository=$3
     if [ -n "$branch" ]; then
         branch_option="--branch $branch"
     fi
     workspace=$(workspace_dir $project)
     repository=$(project_config $project repository)
     rm -rf $workspace
     mkdir -p $workspace
-    result=$(git clone --depth=20 --no-tags $branch_option $repository $workspace 2>&1 || exit $?)
+    if [ "$local_repository" ]; then
+        echo "Cloning from local repository $local_repository"
+        repo=$local_repository
+    else
+        repo=$repository
+    fi
+    result=$(git clone --depth=20 --no-tags $branch_option $repo $workspace 2>&1 || exit $?)
```

Comment:
I initially thought about it. But repository is  a global variable used also in other parts of the script. I was concerned it could introduce subtle bugs the reassignment

Reply:
Nah don't worry we set `repository` it a few lines before this anyway

Reply:
ok then